### PR TITLE
Python Yarn script update

### PR DESCRIPTION
**Pull request by jenniew**
## What changes were proposed in this pull request?

Update example script of running BigDL in Yarn environment.

## How was this patch tested?
tested in yarn env.

### CODE REVIEW

**Comment by hkvision** [comment]
Need to update https://github.com/intel-analytics/BigDL/blob/master/pyspark/python_package/README.md as well for `VENV_HOME`? @jenniew

**Comment by jason-dai** [comment]
Also update the python user guide?

**Comment by jenniew** [comment]
already updated https://github.com/intel-analytics/BigDL/blob/master/pyspark/python_package/README.md

**Comment by hkvision** [comment]
LGTM

File: pyspark/python_package/README.md [hunk 1]
```diff
@@ -55,10 +56,10 @@ Refer to python_submit_yarn_cluster.sh.example to run lenet example on yarn clus
...
     --jars ${BigDL_JAR_PATH} \
     --py-files ${PYTHON_API_PATH} \
-    --archives venv.zip \
+    --archives ${VENV_HOME}/venv.zip \
     --conf spark.driver.extraClassPath=bigdl-VERSION-jar-with-dependencies.jar \
     --conf spark.executor.extraClassPath=bigdl-VERSION-jar-with-dependencies.jar \
-    ${BigDL_HOME}/pyspark/dl/models/lenet/lenet5.py
+    ${BigDL_HOME}/pyspark/bigdl/examples/lenet/lenet.py
  ```
 details can be found at: [LeNet5](https://github.com/intel-analytics/BigDL/tree/master/pyspark/dl/models/lenet/README.md).
```
Comment (hkvision) [inline pyspark/python_package/README.md]: Also need to update this link   `dl` -> `bigdl`

**Comment by jenniew** [comment]
python user guide only provides link to here.

**Pull request merged**